cds-snc/cppd-medical-report-sails
# Move route helper out to policy

## What this does
I noticed that the route helper wasn't always returning the path in the current language, depending on where it was called. For example, calling `route('personal')` from a nunjucks template was returning `/en/personal` no matter what your current locale was.

I assume this was due to the timing of when the helper was being attached to `res.locals.`

To fix, I moved the route helper out to a policy file where I am able to load it after the localization policy fires.

I believe I had it configured this way once before, but was unsure about using "policies" this way. It seems this is an appropriate use afterall, so this is fine.

## To test

Maybe easier to see if you run this test twice- once on Master, then again on this branch, to see the different result:

- Navigate to `/en/start`
- Click `Let's go!`
- It takes you to the english `/en/personal` page

- Navigate to `/fr/debut`
- Click 'Allons-y`
- It takes you to the french `/fr/personnel` page (this is the step that was broken in Master)
